core/iwasm/include/wasm_export.h

```diff
@@ -63,6 +63,30 @@ struct WASMModuleCommon;
 typedef struct WASMModuleCommon *wasm_module_t;
 #endif
 
+typedef enum
+{
+  WASM_IMPORT_EXPORT_KIND_FUNC,
+  WASM_IMPORT_EXPORT_KIND_GLOBAL,
+  WASM_IMPORT_EXPORT_KIND_TABLE,
+  WASM_IMPORT_EXPORT_KIND_MEMORY,
+} wasm_import_export_kind_t;
+
+typedef struct wasm_import_info_t
+{
+    const char * module_name;
+    const char * name;
+    wasm_import_export_kind_t kind;
+    bool linked;
+} wasm_import_info_t;
+
+typedef struct wasm_export_info_t
+{
+    const char * name;
+    wasm_import_export_kind_t kind;
+} wasm_export_info_t;
```

Comment:
for me personally, *export type* is more accurate than *export info*.

Reply:
Can you clarify please? Since the _t extension means "type", do you prefer that it's named wasm_export_t or wasm_export_type_t?